serpextract/serpextract.py

```diff
@@ -506,23 +490,26 @@ def get_all_query_params():
 
     return list(all_params)
 
-
+qs_params = defaultdict(list)
```

Comment:
This should start with `_`, we don't want this cache to be considered part of the public API.